src/Visualization/Python/GenerateVideo.py

```diff
@@ -0,0 +1,312 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import click
+import glob
+import logging
+from paraview.simple import *
+
```

Comment:
A lot of this will depend on the ParaView version. Can you check the version and add a warning message if the version isn't the one you used? We will inevitably need to have `if-else` checks for different versions.